templates/SilverStripe/Admin/Includes/BrowserWarning.ss

```diff
@@ -0,0 +1,31 @@
+<!--[if lte IE 9]>
```

Comment:
are you sure about not including the css file for ie10? as above I saw `version <= 10`

Reply:
Dang! I was so focused on working on the workaround for IE 9 and below. Good catch!